packages/server/start/routes.ts

```diff
@@ -28,71 +28,85 @@ Route.post('/search/users', 'SearchUsersController.searchUsers');
 
 Route.get('/ping', () => console.log('pong'));
 
-/// Temporal Routes ///
+Route.get('/proxy-places-api/*', 'PlacesApisController.proxyPlacesAPIRequest');
+
+/// Temporal MTV Routes ///
+
+const MTV_TEMPORAL_LISTENER = `/temporal/mtv`;
```

Comment:
I think it would be better to use AdonisJS idiomatic way to _group_ routes: https://docs.adonisjs.com/guides/routing#prefix-routes.
That way we do less handmade work.

Reply:
Done

Reply:
Perfect!